hack/commit-release.sh

```diff
@@ -19,7 +19,8 @@
 
 git fetch --tags
 tagcommit=$(git rev-list --tags --max-count=1)
-current=$(git describe --tags $tagcommit | sed 's/v//')
+#current=$(git describe --tags $tagcommit | sed 's/v//')
+current="1.2.14"
```

Comment:
This shouldn't be needed
You usually fix it by `git fetch upstream` to get latest tags locally

Reply:
Good catch!
Fixed, thanks.